src/azmi-main/blob/SetBlob.cs

```diff
@@ -54,10 +53,12 @@ public List<string> Execute(object options)
         // Execute SetBlob
         //
 
-        public static string Execute(string filePath, string blobUri, string identity = null, bool force = false)
```

Comment:
Two changes here:
- removed static - all other methods are not static
- added optional IBlobClient, used in testing

Below in code instead of `=`, I used `??=` (assign only if null), again functionality used in testing